app/controllers/meta.controller.js

```diff
@@ -4,6 +4,7 @@ import Constants from '../config/constants';
 class MetaController extends BaseController {
   index(req, res) {
 		res.json({
+      name: "learn2018-be",
```

Comment:
@siddharthchavan :
Using hard coded text like this can be problematic when dealing with large code bases.

For instance, what if our API returns the "name" field in every response? First, we'd have to add the "name" field to every response controller. But what if that name changed down the line? 

We'd have to edit that name for how ever many responses we had. Right now we just have around 10 calls, which is already a hassle to change. Imagine a larger company that may have 100s of calls. That would be a huge time sink and hassle.

If you see below, the "version" field gets that value from somewhere called Constants. Trace back where the "version" field comes from until you get to a file called package.json. You should see where to get the name: "learn2018-be" from then.